service_invocation/csharp/http/checkout/Program.cs

```diff
@@ -1,24 +1,21 @@
-﻿using System.Text;
-using System.Text.Json;
-using System.Text.Json.Serialization;
+﻿using Dapr.Client;
 
-var baseURL = (Environment.GetEnvironmentVariable("BASE_URL") ?? "http://localhost") + ":" + (Environment.GetEnvironmentVariable("DAPR_HTTP_PORT") ?? "3500");
+using var client = new DaprClientBuilder().Build();

```

Comment:
```suggestion
using var client = DaprClient.CreateInvokeHttpClient();
```
Hi - I think the only change @msfussell suggests in #888 is this exact one here, where you create the *httpclient from the factory function, instead of a generic 'new' one, which still needs the appid and such set.  We should also decide if we pass in the appid explicitly, e.g. `DaprClient.CreateInvokeHttpClient("order-processor");` or if we can pick it up savely from the env variable set by the Dapr CLI.  

All the other changes of using the Dapr client to do invoke's is not what we want.  We want to use the raw HttpClient type and not the DaprClient SDK type.

Reply:
Thanks for your explanation.  However it seems that not all SDKs have such a method. Does it mean that I only need to modify the csharp sample?